src/utils/constants.ts

```diff
@@ -1,13 +1,48 @@
 export const jsonConfigFileStore = 'config.json';
+export const jsonTokenFileStore = 'tokens.json';
 export const zendeskIcon = 'https://raw.githubusercontent.com/mattermost/mattermost-app-zendesk/main/assets/zendesk.svg';
+export const zendeskClientID = 'mattermost_zendesk_app';
+export const formTextAreaMaxLength = 1024;
 
-const zendesk = {
+const mattermostPaths = {
+};
+
+const zendeskPaths = {
+    OAuthAuthorizationURI: '/oauth/authorizations/new',
+    OAuthAccessTokenURI: '/oauth/tokens',
+    APIVersion: '/api/v2',
+};
+
+const appPaths = {
+    ManifestPath: '/manifest.json',
+    InstallPath: '/install',
+
+    OAuthCompletePath: '/oauth/complete',
+
+    BindingsPath: '/bindings',
+    BindingPathCreateForm: '/create',
+    BindingPathConnect: '/connect',
+    BindingPathDisconnect: '/disconnect',
+};
+
+export const routes = {
+    zd: zendeskPaths,
```

Comment:
maybe we should use this naming across the codebase: like zendesk -> zd, use mm as a prefix to the MM entities and maybe keep app entities with no "app" prefix (like root_url) since this is the app codebase.

Reply:
aside from file and folder names, I made the following substitutions throughout.

`zendesk` -> `zd`
`mattermost` -> `mm`